warehouse/query-core/src/test/java/datawave/query/CompositeFunctionsTest.java

```diff
@@ -782,6 +782,22 @@ public void testFilterFunctionsInvalidatedByIndexOnlyFields() throws ParseExcept
         }
     }
 
+    @Test
+    public void test() throws Exception {
```

Comment:
rename to something like test index only rewrite. using the 'plan' end point would allow you to verify the rewritten filter function.